pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -30,19 +30,20 @@ var (
 	ProductsStage         StageName = "products"
 	SolutionExplorerStage StageName = "solution-explorer"
 
-	ProductAMQStreams          ProductName = "amqstreams"
-	ProductAMQOnline           ProductName = "amqonline"
-	ProductSolutionExplorer    ProductName = "solution-explorer"
-	ProductRHSSO               ProductName = "rhsso"
-	ProductRHSSOUser           ProductName = "rhssouser"
-	ProductCodeReadyWorkspaces ProductName = "codeready-workspaces"
-	ProductFuse                ProductName = "fuse"
-	ProductFuseOnOpenshift     ProductName = "fuse-on-openshift"
-	Product3Scale              ProductName = "3scale"
-	ProductNexus               ProductName = "nexus"
-	ProductLauncher            ProductName = "launcher"
-	ProductUps                 ProductName = "ups"
-	ProductMonitoring          ProductName = "monitoring"
+	ProductAMQStreams             ProductName = "amqstreams"
+	ProductAMQOnline              ProductName = "amqonline"
+	ProductSolutionExplorer       ProductName = "solution-explorer"
+	ProductRHSSO                  ProductName = "rhsso"
+	ProductRHSSOUser              ProductName = "rhssouser"
+	ProductCodeReadyWorkspaces    ProductName = "codeready-workspaces"
+	ProductFuse                   ProductName = "fuse"
+	ProductFuseOnOpenshift        ProductName = "fuse-on-openshift"
+	Product3Scale                 ProductName = "3scale"
+	ProductNexus                  ProductName = "nexus"
+	ProductLauncher               ProductName = "launcher"
+	ProductUps                    ProductName = "ups"
+	ProductMonitoring             ProductName = "monitoring"
+	ProductMobileDeveloperConsole ProductName = "mdc"
```

Comment:
:+1: Matches the product id in the [webapp](https://github.com/integr8ly/tutorial-web-app/blob/master/src/product-info.js#L50).